### PR TITLE
Bump hugo to v0.81.0 to solve file size error

### PR DESCRIPTION
**Pull request by jbohanon**
# Description

Hugo v0.81.0 is the first release which includes the fix for a docs gen issue we've been running into:
```log
Error: Error building site: "/home/runner/work/gloo/gloo/docs/gloo-temp/docs/content/reference/security-updates/enterprise/_index.md:12:1": failed to render shortcode "readfile": failed to process shortcode: "/home/runner/work/gloo/gloo/docs/gloo-temp/docs/themes/hugo-theme-soloio/layouts/shortcodes/readfile.html:11:17": execute of template failed: template: shortcodes/readfile.html:11:17: executing "shortcodes/readfile.html" at <readFile>: error calling readFile: file "static/content/glooe-security-scan-3.docgen" is too big
```

The fixed version of Hugo simply removes the arbitrary and hard-coded 1mb filesize limit.

# Checklist:

- [ ] I included a concise, user-facing changelog (for details, see https://github.com/solo-io/go-utils/tree/main/changelogutils) which references the issue that is resolved.
- [ ] If I updated APIs (our protos) or helm values, I ran `make -B install-go-tools generated-code` to ensure there will be no code diff
- [ ] I followed guidelines laid out in the Gloo Edge [contribution guide](https://docs.solo.io/gloo-edge/latest/contributing/)
- [ ] I opened a draft PR or added the work in progress label if my PR is not ready for review
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: docs/Makefile [hunk 1]
```diff
@@ -1,17 +1,12 @@
 # Note to developers/testers
 # >> to run host the docs locally, do the following
+# ensure you have hugo installed e.g. go install github.com/gohugoio/hugo@v0.81.0
 # cd to the docs dir
 # make serve-site -B
 
-# Additional steps are required for darwin/arm64 (M1 macOS)
-# build hugo v0.69.2:
-# https://github.com/gohugoio/hugo must be built from commit ec9dcf304849ded11cea70ab9ad46e0a0a4615b2 and installed on $PATH
```
Comment (jenshu) [inline docs/Makefile:8]: is this not needed anymore?

Comment (jbohanon) [reply docs/Makefile:8]: Nope. As of version [v0.69.2](https://github.com/gohugoio/hugo/releases/tag/v0.69.2), hugo was not publishing darwin/arm64 assets, but by [v0.81.0](https://github.com/gohugoio/hugo/releases/tag/v0.81.0) they were, so mac arm users no longer have to build from source on a specific commit; they can do `go install github.com/gohugoio/hugo@v0.81.0`

**Review by gunnar-solo — APPROVED** [review]

**Review by jenshu — APPROVED** [review]

**Pull request merged**